templates/base.html

```diff
@@ -4,7 +4,9 @@
   <head>
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1">
-    <title>{{ config.title }}</title>
+    <title>
+      {% block title %}{% if page.title %}{{ page.title }} | {% endif %} {{ config.title }}{% endblock %}
+    </title>
```

Comment:
This is a little different from what I would have preferred, but I couldn't get a simple `{% set title = "News" %}` in the template and `{% if title %}{{title}}{% elif page.title %}` etc here to work.